### ISSUE TITLE
GraphQL error on page update

### ISSUE THREAD

**Issue by Spoutnik87** [Requarks/wiki#1260]
**Describe the bug**
I get a `GraphQL error: Cannot return null for non-nullable field ResponseStatus.slug.` error message when i'm trying to save a change on an existing page or when i'm trying to create a new page. Changes are not synced with my git repository and pages are not rerendered but there is some local cache that keep the changes.

**To Reproduce**
1. Create a new page.
2. Save it.
3. See error.

**Expected behavior**
Completing the save process and syncing with git repository + rerendering the page.

**Host Info (please complete the following information):**
 - OS: FreeBSD 11.2
 - Wiki.js version: 2.0.12
 - Database engine: PostgreSQL 11.5

**Additional context**
I imported the data and users from a wiki.js 1.0 installation with an existing git repository.

**Comment by Spoutnik87** [comment]
If it can help, when I save, my client send a request with the following input : 
```json
{
   "0":{
      "extensions":{

      },
      "operationName":null,
      "query":"mutation ($id: Int!, $content: String, $description: String, $editor: String, $isPrivate: Boolean, $isPublished: Boolean, $locale: String, $path: String, $publishEndDate: Date, $publishStartDate: Date, $tags: [String], $title: String) {\n  pages {\n    update(id: $id, content: $content, description: $description, editor: $editor, isPrivate: $isPrivate, isPublished: $isPublished, locale: $locale, path: $path, publishEndDate: $publishEndDate, publishStartDate: $publishStartDate, tags: $tags, title: $title) {\n      responseResult {\n        succeeded\n        errorCode\n        slug\n        message\n        __typename\n      }\n      __typename\n    }\n    __typename\n  }\n}\n",
      "variables":{
         "content":"# Header\nYour content here",
         "description":"",
         "editor":"markdown",
         "id":20,
         "isPrivate":false,
         "isPublished":true,
         "locale":"fr",
         "path":"test",
         "publishEndDate":"",
         "publishStartDate":"",
         "tags":[

         ],
         "title":"Test"
      }
   }
}
```
And the response is :
```json
{
   "0":{
      "errors":[
         {
            "message":"Cannot return null for non-nullable field ResponseStatus.slug.",
            "locations":[
               {
                  "line":7,
                  "column":9
               }
            ],
            "path":[
               "pages",
               "update",
               "responseResult",
               "slug"
            ],
            "extensions":{
               "code":"INTERNAL_SERVER_ERROR",
               "exception":{
                  "stacktrace":[
                     "Error: Cannot return null for non-nullable field ResponseStatus.slug.",
                     "    at completeValue (/root/wiki/node_modules/graphql/execution/execute.js:560:13)",
                     "    at completeValueCatchingError (/root/wiki/node_modules/graphql/execution/execute.js:495:19)",
                     "    at resolveField (/root/wiki/node_modules/graphql/execution/execute.js:435:10)",
                     "    at executeFields (/root/wiki/node_modules/graphql/execution/execute.js:275:18)",
                     "    at collectAndExecuteSubfields (/root/wiki/node_modules/graphql/execution/execute.js:713:10)",
                     "    at completeObjectValue (/root/wiki/node_modules/graphql/execution/execute.js:703:10)",
                     "    at completeValue (/root/wiki/node_modules/graphql/execution/execute.js:591:12)",
                     "    at completeValue (/root/wiki/node_modules/graphql/execution/execute.js:557:21)",
                     "    at completeValueCatchingError (/root/wiki/node_modules/graphql/execution/execute.js:495:19)",
                     "    at resolveField (/root/wiki/node_modules/graphql/execution/execute.js:435:10)",
                     "    at executeFields (/root/wiki/node_modules/graphql/execution/execute.js:275:18)",
                     "    at collectAndExecuteSubfields (/root/wiki/node_modules/graphql/execution/execute.js:713:10)",
                     "    at completeObjectValue (/root/wiki/node_modules/graphql/execution/execute.js:703:10)",
                     "    at completeValue (/root/wiki/node_modules/graphql/execution/execute.js:591:12)",
                     "    at /root/wiki/node_modules/graphql/execution/execute.js:492:16",
                     "    at process._tickCallback (internal/process/next_tick.js:68:7)"
                  ]
               }
            }
         }
      ],
      "data":{
         "pages":{
            "update":null,
            "__typename":"PageMutation"
         }
      }
   }
}
```

**Comment by Spoutnik87** [comment]
I managed to solve this issue. It's relative to my setup.
I close the issue.

**Comment by uropro** [comment]
Hi Spoutnik87
I run the same wiki.js version on FreeBSD 12.0 and encounter the same error messages as you.
What solved your problem?
Many thanks in advance.

**Comment by Spoutnik87** [comment]
Hi,
I had two problems with my setup. I inadvertently had multiple instances of wikijs running in the same time and I tried to run wikijs in background by using nohup. ( the command was : `nohup node server &`) I had the issue when I left the shell. I guess something goes wrong when the process is detached.

I solved this issue by using [PM2](https://pm2.keymetrics.io/) and just starting the process with the command `pm2 start server`.
I hope this can help you to solve this issue.
